lib/yi_appium_caps_util.rb

```diff
@@ -122,9 +134,35 @@ def update_ios_caps (output_data)
             ipcounter = 999
           end
         end
-        File.delete('iplog.txt')
-        #Replace value of udid
-        output_data['caps']['youiEngineAppAddress'] = ip
+        # Get the device name before deleting the file
+        deviceName = %x[grep -Eo "deviceName =.*" logs.txt| head -1|cut -d " " -f 3-|tr -d '"'|tr -d '\n']
+        platformVersion = %x[grep -Eo "platformVersion =.*" logs.txt| head -1|awk '{print $3}'| tr -d '"'|tr -d '\n']
+        puts 'DeviceName: ' + deviceName
+        puts 'platformVersion: ' + platformVersion
+        File.delete('logs.txt')
+        #Replace value of ip if found
+        if (ip != '') then 
+          output_data['caps']['youiEngineAppAddress'] = ip 
+        else 
+          puts "Update ip address manually"
+        end
+        output_data['caps']['deviceName'] = deviceName
+        output_data['caps']['platformVersion'] = platformVersion
+
+        xcodeBuildVersion = %x[xcodebuild -version | head -1 | awk '{print $2}']
+
+        # Add the xcodeConfigFile in the caps if dealing with iOS 10+
+        if (platformVersion.to_f>=10) then
+          output_data['caps']['xcodeOrgId'] = 'W4E9HL2DXS'
```

Comment:
Yea this is next. I have something working locally I want to try it out before updating the PR